src/test/comment/postComments.test.js

```diff
@@ -0,0 +1,255 @@
+import chai from 'chai';
+import chaiHttp from 'chai-http';
+import sinon from 'sinon';
+import app from '../../index';
+import commentController from '../../controllers/comment.controller';
+import commentMiddleware from '../../middlewares/comment.middleware';
+
+const { expect } = chai;
+
+chai.use(chaiHttp);
+
+let myToken;
+
+before((done) => {
```

Comment:
I think it will be a lot easier and faster if you would just import `jwt.service` and call its `generateToken` method on an object like `{ id: 2, role: 'requester' }` to obtain a token. What do you think?